### PR TITLE
Removing 'Hacks' for a global quantities table 'Global' 

### PR DESCRIPTION
**Pull request by KSchoenleber**
In this I am removing the 'Hacks' worksheet in the input file, replacing it by a similar sheet for global quantities. Furthermore, the property 'wacc' is then moved into this sheet and removed from the 'process', 'transmission' and 'storage' sheet.

I also changed the sorting of the worksheets to a less historical and in my opinion more clear order and changed the color of the 'DSM' sheet.

Overall I think these changes make the input a lot clearer!

Potential problems: - Backwards compatibility
                              - Slightly lower functionality since 'wacc' is now set identical for everything

### CODE REVIEW

**Comment by maledo** [comment]
Good idea, but why is should we put WACC as a global input? Especially if we model on higher scales (different countries) WACC migth differ from country to country.

File: runme.py [hunk 1]
```diff
@@ -23,8 +23,8 @@ def scenario_stock_prices(data):
 
 def scenario_co2_limit(data):
     # change global CO2 limit
-    hacks = data['hacks']
-    hacks.loc['Global CO2 limit', 'Value'] *= 0.05
+    glob = data['global']
```
Comment (maledo) [inline runme.py]: Is 'global' instead of 'glob' really to long?

**Comment by KSchoenleber** [comment]
I thought about that too. I find the placement of WACC at the moment a little much. A good way is to place it in the 'Site' sheet instead making it a property of a given site. I will do that. Wait for the update!

**Comment by maledo** [comment]
But might it not also differ on the type of technology?

**Comment by KSchoenleber** [comment]
I really think it is more of a regional thing, having to do with the local investment environment.

**Comment by KSchoenleber** [comment]
I guess I will revert it completely as you suggest. Otherwise transmissions might become a real headache!

File: runme.py (hunk 1 above)
Comment (KSchoenleber) [reply runme.py]: No but the word 'global' calls a function in python and can thus not be set unless it is a string

**Comment by KSchoenleber** [comment]
So now it should all be as requested!

File: runme.py (hunk 1 above)
Comment (maledo) [reply runme.py]: makes sense :)

**Pull request merged**